src/core_atmosphere/CMakeLists.txt

```diff
@@ -76,12 +84,17 @@ set(ATMOSPHERE_CORE_PHYSICS_WRF_SOURCES
     module_sf_noahlsm.F
     module_sf_noahlsm_glacial_only.F
     module_sf_oml.F
+    module_sf_ruclsm.F
```

Comment:
we do not need module_sf_ruclsm.F, it is replace with module_ruc_land.F90 and module_ruc_ice.F90

Reply:
@tanyasmirnova Thanks for the comment. 
Do you suggest I remove this file from the CMakeLists.txt?

Reply:
Done. Thanks, Tanya!